blueocean-dashboard/src/main/js/components/Runs.jsx

```diff
@@ -74,7 +75,9 @@ export default class Runs extends Component {
                 {id}
             </td>
             <td><CommitHash commitId={commitId} /></td>
-            {showBranchCol && <td>{decodeURIComponent(pipeline)}</td>}
+            <IfCapibility dataClass={pipelineClass} capability="io.jenkins.blueocean.rest.model.BlueMultiBranchPipeline" >
+                <td>{decodeURIComponent(pipeline)}</td>
+            </IfCapibility>
```

Comment:
Right, it was my plan to do this, its just a wip that i commited so i could get some help